addons/web/static/src/views/list/list_renderer.js

```diff
@@ -1266,6 +1266,11 @@ export class ListRenderer extends Component {
 
         const hotkey = getActiveHotkey(ev);
 
+        if (hotkey === 'control+enter' && this.props.onOpenFormView){
+            this.isX2Many && record.isNew ? this.displaySaveNotification() : this.props.onOpenFormView(record)
+            return;
+        }
+
```

Comment:
Hello @hatr-odoo code is not linted properly.

Also will there is any condition to open formview from x2many ?

Reply:
Hello @vivp-odoo,
I have linted and updated the code.